oat-sa/extension-parcc-tei
# Fix/tr 4990/solar tr support

Related to: https://oat-sa.atlassian.net/browse/TR-4990

Fixed issue with handlebars 
Fixed issue with ids

How to test without Solar Delivery:

- create items with PCIs
- create test
- install https://github.com/oat-sa/extension-tao-test-preview-ui-loader
- preview test, all PCIs should be rendered
